docs/source/algorithms/EnggFitPeaks-v1.rst

```diff
@@ -36,14 +36,13 @@ the Mantid :ref:`ConvertUnits <algm-ConvertUnits>`. See also
 :ref:`EnggFitDIFCFromPeaks <algm-EnggFitDIFCFromPeaks>`.
 
 This algorithm currently fits (single) peaks of type
-:ref:`Back2BackExponential <func-Back2BackExponential>`. Other
+:ref:`Back2BackExponential <func-BackToBackExponential>`. Other
 alternatives might be added as optional in the future (for example the
 simpler :ref:`Gaussian <func-Gaussian>` or the more complex
-:ref:`Bk2BkExpConvPV <func-Bk2BkExpConvPV` or :ref:`IkedaCarpenterPV
-<func-IkedaCarpenterPV>`). To produce an initial guess for the peak
-function parameters this algorithm uses the :ref:`FindPeaks
-<algm-FindPeaks>` algorithm starting from the expected peaks list
-given in the inputs.
+:ref:`Bk2BkExpConvPV <func-NeutronBk2BkExpConvPVoigt>` or :ref:`IkedaCarpenterPV <func-IkedaCarpenterPV>`). To
```

Comment:
Good catch. I thought I read in a file that Bk2BkExpConvPV had just been renamed but I must have misunderstood.

I think you are correct, it should not have a link if one does not exist. I'll take it out.